src/job/job.spec.js

```diff
@@ -49,8 +66,10 @@ describe("job", () => {
         const job = await xo.call("job.get", { id });
         expect(job).toMatchSnapshot({
           id: expect.any(String),
+          paramsVector: expect.any(Object),
           userId: expect.any(String),
         });
+        testParamsVector(job.paramsVector, "test-snapshot");
```

Comment:
use `defaultJob.paramsVector` instead.

Reply:
We must test the job created with the `defaultJob` parameters. Why use `defaultJob.paramsVector` in this test?

Reply:
Your first sentence responds to the question, No?

Reply:
We need to test the correspondence between the created job and the input params which are the `defaultJob` params.